packages/commonwealth/server/CommonwealthConsumer/messageProcessors/snapshotConsumer.ts

```diff
@@ -10,6 +10,14 @@ export async function processSnapshotMessage(
 ) {
   const { space, id, title, body, choices, start, expire } = data;
 
+  const eventType = data.event;
+
+  if (!space && eventType !== 'proposal/deleted') {
```

Comment:
Why does eventType matter here?

Reply:
in the event that its a deletion, we expect there not to be a space. And that is handled by our processing. The other event types SHOULD have a space, but sometimes do not (see discussion in slack channel).